persistent/Database/Persist/Sql/Orphan/PersistUnique.hs

```diff
@@ -110,11 +117,34 @@ instance PersistUniqueRead SqlBackend where
         t = entityDef $ dummyFromUnique uniq
         toFieldNames' = toList . fmap snd . persistUniqueToFieldNames
 
+    existsBy uniq = do
```

Comment:
Just realized that maybe this could be simplified to something like
```hs
existsBy uniq = exists $ persistUniqueToValues uniq
```
if
* we had access to `PersistQueryRead` here, for `exists`
* and the signature of `exists`, or possibly an inner helper, was adapted to also accept `[PersistValue]`, which is what is obtained in `exists` already after calling `getFiltsValues`